commit 7956536a3d78ba0ef8ec990651b315664ed70f90
Author: popcornmix <popcornmix@gmail.com>
Date:   Mon Mar 21 15:38:38 2016 +0000

    dwc_otg: Enable the hack for Split Interrupt transactions by default
    
    dwc_otg.fiq_fsm_mask=0xF has long been a suggestion for users with audio stutters or other USB bandwidth issues.
    So far we are aware of many success stories but no failure caused by this setting.
    Make it a default to learn more.
    
    See: https://www.raspberrypi.org/forums/viewtopic.php?f=28&t=70437
    
    Signed-off-by: popcornmix <popcornmix@gmail.com>

diff --git a/drivers/usb/host/dwc_otg/dwc_otg_driver.c b/drivers/usb/host/dwc_otg/dwc_otg_driver.c
index 95edadf..cb060a7 100644
--- a/drivers/usb/host/dwc_otg/dwc_otg_driver.c
+++ b/drivers/usb/host/dwc_otg/dwc_otg_driver.c
@@ -247,7 +247,7 @@ bool fiq_fsm_enable = true;
 //Bulk split-transaction NAK holdoff in microframes
 uint16_t nak_holdoff = 8;
 
-unsigned short fiq_fsm_mask = 0x07;
+unsigned short fiq_fsm_mask = 0x0F;
 
 /**
  * This function shows the Driver Version.
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1207_7956536a3d78ba0ef8ec990651b315664ed70f90.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1207_7956536a3d78ba0ef8ec990651b315664ed70f90.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
